### PR TITLE
Remove Joda time

### PR DESCRIPTION
**Pull request by PenghaiZhang**
PR #1022 
Joda Time was used in the wizard control of Youtube and Echo360.

Removing Joda from the Youtube one is simple, but doing that for  Echo360 is harder because we are unable to test it at this moment.

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/echo/EchoUtils.java [hunk 1]
```diff
@@ -51,29 +47,10 @@
...
-            .appendSeparator(ampersand)
-            .appendSeconds()
-            .appendSuffix(ds("second"), ds("seconds"))
-            .toFormatter();
-
-    return pf.print(period).trim();
+    LocalDateTime now = LocalDateTime.now();
+    LocalDateTime plus = now.plus(Duration.ofMillis(duration * 1000));
+    Duration interval = Duration.between(now, plus);
+    return YoutubeUtils.formatDuration(interval.toString());
```
Comment (PenghaiZhang) [inline Source/Plugins/Core/com.equella.core/src/com/tle/web/echo/EchoUtils.java]: Current change just simply reuses the format used in the Youtube control for two reasons: 
1. It takes more efforts to convert the PeriodFormatter which Java 8 has no similar API provided.
2. According to Charlie, we can't test Echo360 at this moment. Without seeing how this really works, I wonder if we can rework the formatting later on.

Comment (edalex-ian) [reply Source/Plugins/Core/com.equella.core/src/com/tle/web/echo/EchoUtils.java]: Seems fine. I actually wonder if what you've done now address the TODO up on line 48.

**Comment by edalex-ian** [comment]
Aha! The fix you made for the autotest was indeed what I was alluding to when I said:

> I assume you've tested and ensured the resultant format is as it was before.

As that exact bit of code gave me pause. Lucky the autotest picked it up, but something to keep in mind next time. ;)

Go for merge.

**Comment by PenghaiZhang** [comment]
> Aha! The fix you made for the autotest was indeed what I was alluding to when I said:
> 
> > I assume you've tested and ensured the resultant format is as it was before.
> 
> As that exact bit of code gave me pause. Lucky the autotest picked it up, but something to keep in mind next time. ;)
> 
> Go for merge.

Sorry about that. Now I know why I did not test it.  This bit of code gets executed when using Youtube API v2 which has been deprecated. So in my tests I never got this bit running. 
Apologise!

**Pull request merged**